src/openrct2/peep/Peep.h

```diff
@@ -664,27 +664,27 @@ struct Peep : SpriteBase
     uint8_t step_progress;
     union
     {
-        uint16_t mechanic_time_since_call; // time getting to ride to fix
-        uint16_t next_in_queue;
+        uint16_t MechanicTimeSinceCall; // time getting to ride to fix
```

Comment:
It would have been better design by CS if he had combined this with `TimeInQueue` but oh well.